styles.css

```
@@ -0,0 +1,25 @@
+body {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+  #img-div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
```

same here, you can also modify the rest of the code.
```suggestion
  }
  
```